lib/core/test/unit/Cardano/Pool/MetricsSpec.hs

```diff
@@ -288,6 +312,9 @@ prop_trackRegistrations test = monadicIO $ do
                 pure (0, mempty)
             , networkTip =
                 pure header0
+            , staticBlockchainParameters =
+                (block0, mockBlockchainParameters
+                    { getEpochStability = Quantity 42 })
```

Comment:
Question: where does the number 42 come from, and does it have a special meaning?

Even if it's just an arbitrary number, perhaps adding a named constant will help future readers to know that it's just an arbitrary number? (For example, `arbitraryEpochStability` or `mockEpochStability`.)

Reply:
OK fixed.